### PR TITLE
Sistr v1.1.3 new Galaxy wrapper release

### PR DESCRIPTION
**Pull request by kbessonov1984**
This is a new and improved SISTR v1.1.3 Galaxy wrapper that
- provides support for the new `-list-of-serovars` option introduced in this release allowing user to use default or custom list of serovars to check SISTR prediction against
- now all output items in the history get filename added to their name to allow for easier identification especially in cases a `History` has large items
- Updated tests to check for the serovar in the list feature

### CODE REVIEW

File: tools/sistr_cmd/sistr_cmd.xml
Comment (emarinier) [reply tools/sistr_cmd/sistr_cmd.xml:30]: Yeah, I ran this locally and this was the command it made:

```
sistr -i '/tmp/tmprsd3h0ia/files/9/a/9/dataset_9a925318-cf30-4d65-a67d-ff25dc86c75c.dat' '00_0163' -f tab -o sistr-report.tab -p /tmp/tmprsd3h0ia/job_working_directory/000/2/outputs/dataset_420e7d67-6458-456d-8ba4-ccd6ec121b6f.dat -n /tmp/tmprsd3h0ia/job_working_directory/000/2/outputs/dataset_186fc04f-19dd-4074-8570-d9290967021a.dat -a /tmp/tmprsd3h0ia/job_working_directory/000/2/outputs/dataset_74bab77c-1e5f-488b-8610-65378cc89324.dat --run-mash --qc --list-of-serovars --threads "${GALAXY_SLOTS:-1}" -T "${TMPDIR:-/tmp}" -vv
```

It didn't fail, but it create a hanging `--list-of-serovars`. Is that right?

**Review by emarinier — CHANGES_REQUESTED** [review]
I ran it locally through planemo on some test data and didn't see any errors. I'm on vacation for two weeks after today, so I won't be able to approve after further changes. Might want to ask @apetkau to give it a final look over later.

File: tools/sistr_cmd/sistr_cmd.xml
Comment (kbessonov1984) [reply tools/sistr_cmd/sistr_cmd.xml:106]: Yes we can use more general URL, but it might change with different SISTR version in future, so I wanted it to be version specific.

File: tools/sistr_cmd/.shed.yml
Comment (kbessonov1984) [reply tools/sistr_cmd/.shed.yml:5]: This was fixed. Corrected spelling. Thank you

File: tools/sistr_cmd/sistr_cmd.xml
Comment (kbessonov1984) [reply tools/sistr_cmd/sistr_cmd.xml]: Yes the `diff` is a better way. I found that slightly different results are generated depending on the `sistr` installation environment most probably due to slightly allele difference in sorting giving this randomness. This test is not very critical but shows that some sources of variation do exist. I changed to `diff`

**Review by apetkau — APPROVED** [review]

File: tools/sistr_cmd/.shed.yml
Comment (emarinier) [reply tools/sistr_cmd/.shed.yml:5]: I don't think this was changed.

File: tools/sistr_cmd/sistr_cmd.xml
Comment (emarinier) [reply tools/sistr_cmd/sistr_cmd.xml]: What about using the v1.1.3 tag?

https://raw.githubusercontent.com/phac-nml/sistr_cmd/v1.1.3/sistr/data/serovar-list.txt

Comment (kbessonov1984) [reply tools/sistr_cmd/sistr_cmd.xml]: ok, that is the best solution

File: tools/sistr_cmd/.shed.yml
Comment (kbessonov1984) [reply tools/sistr_cmd/.shed.yml]: oh, I changed in the `sistr_cmd.xml` which had identical typo. I now changed in `.shed.yml`

**Comment by kbessonov1984** [comment]
completed all requests

**Review by apetkau — APPROVED** [review]
Looks great. Thanks so much.

**Pull request merged**
